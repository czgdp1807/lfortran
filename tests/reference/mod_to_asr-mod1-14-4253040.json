{
    "basename": "mod_to_asr-mod1-14-4253040",
    "cmd": "lfortran mod --show-asr --no-color {infile}",
    "infile": "tests/interop/mod1-14.mod",
    "infile_hash": "00dfd17351427824dd19adcb6958a8bbbd932547448a184c8ad952bc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "mod_to_asr-mod1-14-4253040.stdout",
    "stdout_hash": "779e3bf0cd88f9e7d08d2c90c2142a2f984f6e450b9046d7fdddd7bf",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}