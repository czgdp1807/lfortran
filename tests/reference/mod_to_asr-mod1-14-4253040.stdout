(TranslationUnit (SymbolTable 1 {f1: (Function (SymbolTable 2 {}) f1 (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f2: (Function (SymbolTable 3 {}) f2 (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f2b: (Function (SymbolTable 4 {}) f2b (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f3: (Function (SymbolTable 5 {}) f3 (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f3b: (Function (SymbolTable 6 {}) f3b (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f4: (Function (SymbolTable 7 {}) f4 (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public), f5b: (Function (SymbolTable 8 {}) f5b (FunctionType [] () GFortranModule Interface () .false. .false. .false. .false. .false. [] [] .false.) [] [] [] () Public)}) [])
